siliconcompiler/core.py

```diff
@@ -513,7 +523,11 @@ def create_cmdline(self,
 
         # Set loglevel if set at command line
         if 'option_loglevel' in cmdargs.keys():
-            self.logger.setLevel(cmdargs['option_loglevel'])
+            log_level = cmdargs['option_loglevel']
+            if isinstance(log_level, list):
```

Comment:
Is this in case multiple nodes set the loglevel?

Reply:
yes, if you set to be different on different steps and indices we have to pick one before the full parsing is complete.

Reply:
Does the actual logging later switch between different granularity levels?
If not we should probably add a little warning that we just picked the first.

Reply:
yes, the logger is reinitialized between nodes to honor these.
This will be corrected later, when we set the loglevel below, this is only the logging level while parsing the values, I'm inclined to not add additional outputs about it. If anything we could pick the highest and use that but I don't think that would be for this PR